### PR TITLE
Support for defined groups

### PR DESCRIPTION
**Pull request by nsharma-git**
# Description

Add support for custom service names

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Testing on Docker setup and QE soak clusters

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: discovery/service/control_center.py [hunk 1]
```diff
@@ -260,6 +287,58 @@ def _build_kerberos_properties(self, service_prop: dict) -> tuple:
...
+        key5 = "confluent.controlcenter.schema.registry.sr-cluster.ssl.truststore.password"
+
+        self.mapped_service_properties.add(key1)
+        self.mapped_service_properties.add(key2)
+        self.mapped_service_properties.add(key3)
+        self.mapped_service_properties.add(key4)
+        self.mapped_service_properties.add(key5)
+
+        if key1 in service_props:
+            return 'all', {'ksql_ssl_enabled': True}
```
Comment (utkarsh5474) [inline discovery/service/control_center.py:340]: Should this be schema_registry_ssl_enabled?

**Review by utkarsh5474 — APPROVED** [review]
nit: lint failures, and below comment
Thanks a lot, Neeraj. We'll need a ticket for Alex too, right?

File: discovery/service/control_center.py (hunk 1 above)
Comment (nsharma-git) [reply discovery/service/control_center.py:340]: Yes. Thanks !

File: discovery/README.md [hunk 2]
```diff
@@ -22,30 +31,96 @@ cd <some_path>/ansible_collections/confluent/platform
...
+vars:
+  ansible_connection: ssh
+  ansible_user: centos
+  ansible_become: true
+  ansible_ssh_extra_args: -o StrictHostKeyChecking=no
+  ansible_ssh_private_key_file: ~/Work/keys/muckrake.pem
+
+hosts:
+  all:
+    - ec2-35-85-206-158.us-west-2.compute.amazonaws.com
```
Comment (utkarsh5474) [inline discovery/README.md:45]: Unrelated note:
@nsharma-git 
Should we use some dummy name and not actual keyname and ec2 addresses? 
Atleast, for cp-ansible docs if not for the readme.

**Pull request merged**